samples/Foundatio.ReceiverConsole/Receiver.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Text;
+using System.Threading;
+using Foundatio.Messaging;
+using Foundatio.Queues;
+using System.Threading.Tasks;
+using Foundatio.Tests.Utility;
+namespace Foundatio.ReceiverConsole {
+    class Receiver {
+        private async Task TestTopic() {
+            IMessageBus messageBus =
+                new AzureServiceBusMessageBus(new AzureServiceBusMessageBusOptions() {
```

Comment:
If we update to the latest foundatio 6, it brings in the logging changes and we've migrated everything to the new cross plat configuration system). We can accept the pr as is but if you wanted to tackle that it's pretty fun to play with and learn :)